File: Front/src/app/Models/Contactos.ts
import { Usuarios } from './Usuarios';

export class Contactos{
    constructor(
        public Id:number,
        public ContactName:string,
        public ContactLastName:string,
        public TelephoneNumber:string,
        public UsuarioId:number,
        public Usuario:Usuarios
    ){

    }
}

File: Back/AgendaWeb/Models/Contactos.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace AgendaWeb.Models
{
    public class Contactos
    {
        [Key]
        public int Id { get; set; }
        public string ContactName { get; set; }
        public string ContactLastName { get; set; }
        public string TelephoneNumber { get; set; }
        [ForeignKey("usuario")]
        public int UsuarioId { get; set; }
        public Usuarios Usuario { get; set; }

    }
}
